src/App.js

```diff
@@ -525,6 +537,89 @@ export const App = () => {
 		}
 	};
 
+	const onDeposit = async (neoBalanceTemp) => {
+		if (depositQuantity * 1e8 >= 1 && depositQuantity * 1e8 <= neoBalanceTemp) {
```

Comment:
Can it be fractional (1.5 GAS)? We accept it here.

Reply:
yes, it can